src/components/ComputeModal.js

```diff
@@ -1006,19 +1009,22 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
       div({ style: { ...gridStyle, gridTemplateColumns: '0.25fr 8.5rem 1fr 5.5rem 1fr 5rem', marginTop: '1.5rem' } }, [
         h(IdContainer, [
           id => div({ style: { gridColumnEnd: 'span 3' } }, [
-            label({ htmlFor: id, style: computeStyles.label }, ['Location']),
+            label({ htmlFor: id, style: computeStyles.label }, ['Location ']),
+            betaVersionTag,
             h(InfoBox, { style: { marginLeft: '0.5rem' } }, [
-              'Cloud environments run in the same region as the workspace bucket and cannot be changed at this time.'
+              'Cloud environments run in the same region as the workspace bucket and can be changed as a beta feature.'
             ]),
             div({ style: { marginTop: '0.5rem' } }, [
               h(Select, {
                 id,
-                isDisabled: true, // Initially releasing regionality supporting only us-central1 region
-                // isDisabled: computeExists, // Can't update location of existing environments
+                // Location dropdown is disabled for:
+                // 1) If editing an existing environment (can't update location of existing environments)
+                // 2) Workspace buckets that are either us-central1 or us multi-regional
+                isDisabled: computeExists || isUSLocation(bucketLocation),
```

Comment:
It feels like we need a tooltip explaining why it is disabled when it is disabled. What do you think?

![image](https://user-images.githubusercontent.com/5438223/142286887-844a6b35-5dfd-4167-9023-34279a1abb5a.png)

Reply:
I agree, that makes total sense. I added these two messages depending on why its disabled:
>Cannot update the location of an existing cloud environment. Delete your cloud environment to create a new one in a different region.

>Currently US workspaces can only have US cloud environments.